Allow user to configure if they should use local file system for review (#349)
pwntester/octo.nvim: README.md

```lua
require"octo".setup({

>   use_localfs = false,                     -- use local fs on right side if in pr branch

Reviewer: ```suggestion
  use_local_fs = false,                     -- use local files on right side of reviews
```
